### DISCUSSION TITLE
无法正常投屏到云视听小电视投屏

### DISCUSSION THREAD

**Discussion by asd1054** [xfangfang/Macast#134] [General]
已安装potplayer插件，nva协议插件，
投屏到云视听小电视投屏时，就会出现
![image](https://user-images.githubusercontent.com/35734032/161428522-87c80f46-d5d1-4682-9afd-c7d730adc3d2.png)

选择其他方式就能投屏，但是设备名字竟然是其他的，比如说我曾经链接过《乐播投屏电视》，所以现在投屏到电脑的名字就是点击这个才能投屏

**Comment by xfangfang** [comment]
> 选择其他方式就能投屏

这里具体是什么含义？

建议一步一步测试，比如先测试DLNA+MPV，然后测试NVA+MPV，最后再测试NVA+Potplayer，不然很难找到问题的原因。

反馈问题最好还是附带log，关于怎么查看log，readme写的很清楚了

**Reply by xfangfang** [reply]
@asd1054  还有其他您认为可以对复现问题、解决问题有帮助的信息也都是需要的，因为软件与插件肯定是在可用的情况下发布出来的，出现问题大概率是您的本地环境存在某个变量与我的环境不同，只是说存在的现象对解决问题是没有多大帮助的。

**Reply by asd1054** [reply]
问题已解决，然而并没有做过任何操作，应该是手机端哔哩哔哩识别问题，当时识别有误


其他方式投屏指：下列画圈地方哦
![Screenshot_20220403_225425](https://user-images.githubusercontent.com/35734032/161433944-02596e49-e2e8-480b-9c4c-bc10682d935f.jpg)

然后成功使用NVA插件投屏视频之后，发现一个小bug，在potplayer没有字幕，连刷好几个视频投屏也没有，在macast文件夹中能够看到字幕文件是实实在在的下载成功了，potplayer也是打开的有ass动态字幕模式，但是也没有字幕，在我手动重新选择字幕之后，就能看到字幕了，在后续的视频投屏中也能正常看到字幕了，望周知

**Reply by xfangfang** [reply]
potplayer插件的readme中并没有表明支持弹幕的内容
